.github/workflows/build.yml

```diff
@@ -75,6 +75,8 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - uses: foundry-rs/foundry-toolchain@v1
+        with:
+          version: nightly-5ac78a9cd4b94dc53d1fe5e0f42372b28b5a7559
```

Comment:
```suggestion
          # Pinning to a version in June 2024, so in case a build fails, our CI doesn't fail
          version: nightly-5ac78a9cd4b94dc53d1fe5e0f42372b28b5a7559
```